tests/layer_tests/ovc_python_api_tests/test_paddle.py

```diff
@@ -97,7 +97,7 @@ class TestMoConvertPaddle(CommonMOConvertTest):
     def test_mo_import_from_memory_paddle_fe(self, create_model, ie_device, precision, ir_version,
                                              temp_dir):
         fw_model, graph_ref, mo_params = create_model(temp_dir)
-        test_params = {'input_model': fw_model, 'use_new_frontend': True}
```

Comment:
Is it really something that is required for ovc? I mean ovc itself doesn't have this parameter, but looks like the test infrastructure may have.